<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usuario</class>
 <widget class="QDialog" name="usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">backgroud-color:#fffff;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 66, 66);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="menu_usuario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 25px;  /* Tamanho da fonte ajustado para um visual mais formal */
    font-weight: bold; /* Pode adicionar para destacar o texto */
    color: white; </string>
    </property>
    <property name="text">
     <string>Menu Usuário</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_editar_usuario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background:rgb(55, 158, 255);
    color: white;               /* Cor do texto: branco */
    font-family: &quot;Times New Roman&quot;;  /* Fonte formal */
    font-size: 18px;            /* Tamanho da fonte */
    font-weight: bold;          /* Negrito para destacar o texto */
    border-radius: 15px;        /* Borda arredondada */
}

QPushButton:hover {
    background-color: #007bff;  /* Cor do fundo quando o mouse está sobre o botão: azul claro */
    color: #333333;             /* Cor do texto ao passar o mouse: cinza escuro */
}

QPushButton:pressed {
    background-color: #003366;  /* Cor do botão ao ser pressionado: azul ainda mais escuro */
    color: #ffffff;             /* Cor do texto ao ser pressionado: branco */
}</string>
    </property>
    <property name="text">
     <string> Editar Conta</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/imagem/imagens/editar_conta.png</normaloff>:/imagem/imagens/editar_conta.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_excluir_usuario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background:rgb(55, 158, 255);
    color: white;               /* Cor do texto: branco */
    font-family: &quot;Times New Roman&quot;;  /* Fonte formal */
    font-size: 18px;            /* Tamanho da fonte */
    font-weight: bold;          /* Negrito para destacar o texto */
    border-radius: 15px;        /* Borda arredondada */
}

QPushButton:hover {
    background-color: #007bff;  /* Cor do fundo quando o mouse está sobre o botão: azul claro */
    color: #333333;             /* Cor do texto ao passar o mouse: cinza escuro */
}

QPushButton:pressed {
    background-color: #003366;  /* Cor do botão ao ser pressionado: azul ainda mais escuro */
    color: #ffffff;             /* Cor do texto ao ser pressionado: branco */
}</string>
    </property>
    <property name="text">
     <string> Excluir Conta</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/imagem/imagens/remover_conta.png</normaloff>:/imagem/imagens/remover_conta.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_metodo_pagamento">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background:rgb(55, 158, 255);
    color: white;               /* Cor do texto: branco */
    font-family: &quot;Times New Roman&quot;;  /* Fonte formal */
    font-size: 18px;            /* Tamanho da fonte */
    font-weight: bold;          /* Negrito para destacar o texto */
    border-radius: 15px;        /* Borda arredondada */
}

QPushButton:hover {
    background-color: #007bff;  /* Cor do fundo quando o mouse está sobre o botão: azul claro */
    color: #333333;             /* Cor do texto ao passar o mouse: cinza escuro */
}

QPushButton:pressed {
    background-color: #003366;  /* Cor do botão ao ser pressionado: azul ainda mais escuro */
    color: #ffffff;             /* Cor do texto ao ser pressionado: branco */
}</string>
    </property>
    <property name="text">
     <string> Formas de Pagamento</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/imagem/imagens/pagamento.png</normaloff>:/imagem/imagens/pagamento.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="nome_usuario">
       <property name="styleSheet">
        <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 15px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: white; </string>
       </property>
       <property name="text">
        <string>Nome do cidadão</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="email_usuario">
       <property name="styleSheet">
        <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 15px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: white; </string>
       </property>
       <property name="text">
        <string>E-mail do cidadão</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="telefone_usuario">
       <property name="styleSheet">
        <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 15px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: white; </string>
       </property>
       <property name="text">
        <string>Telefone</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="cpf_usuario">
       <property name="styleSheet">
        <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 15px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: white; </string>
       </property>
       <property name="text">
        <string>CPF</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tipo_usuario">
       <property name="styleSheet">
        <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 15px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: white; </string>
       </property>
       <property name="text">
        <string>Tipo de Usuário</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="avaliacao_usuario">
       <property name="styleSheet">
        <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 15px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: white; </string>
       </property>
       <property name="text">
        <string>Avaliação</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_menu_servico">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_menu_servico{
	background: transparent;
	border-radius: 30px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/imagem/imagens/menu_servico.png</normaloff>:/imagem/imagens/menu_servico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#info{
	background: transparent;
	border-radius:30px;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/imagem/imagens/info.png</normaloff>:/imagem/imagens/info.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>801</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="btn_salvar_editar">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>470</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   background:rgb(55, 158, 255);
    color: white;               /* Cor do texto: branco */
    font-family: &quot;Times New Roman&quot;;  /* Fonte formal */
    font-size: 20px;            /* Tamanho da fonte */
    font-weight: bold;          /* Negrito para destacar o texto */
    border-radius: 15px;        /* Borda arredondada */
}

QPushButton:hover {
    background-color: #007bff;  /* Cor do fundo quando o mouse está sobre o botão: azul claro */
    color: #333333;             /* Cor do texto ao passar o mouse: cinza escuro */
}

QPushButton:pressed {
    background-color: rgb(255, 0, 4);  /* Cor do botão ao ser pressionado: azul ainda mais escuro */
    color: #ffffff;             /* Cor do texto ao ser pressionado: branco */
}</string>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancelar_editar">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   background:rgb(55, 158, 255);
    color: white;               /* Cor do texto: branco */
    font-family: &quot;Times New Roman&quot;;  /* Fonte formal */
    font-size: 20px;            /* Tamanho da fonte */
    font-weight: bold;          /* Negrito para destacar o texto */
    border-radius: 15px;        /* Borda arredondada */
}

QPushButton:hover {
    background-color: #007bff;  /* Cor do fundo quando o mouse está sobre o botão: azul claro */
    color: #333333;             /* Cor do texto ao passar o mouse: cinza escuro */
}

QPushButton:pressed {
    background-color:rgb(34, 255, 0);  /* Cor do botão ao ser pressionado: azul ainda mais escuro */
    color: #ffffff;             /* Cor do texto ao ser pressionado: branco */
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="senha_editar">
         <property name="styleSheet">
          <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 19px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: black; </string>
         </property>
         <property name="text">
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="campo_editar_senha">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	/*background:transparent;*/
	border:none;
	/*color:#717072;*/
	border-bottom:1px solid #717072; 
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="nome_editar">
         <property name="styleSheet">
          <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 19px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: black; </string>
         </property>
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="campo_editar_nome">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	/*background:transparent;*/
	border:none;
	/*color:#717072;*/
	border-bottom:1px solid #717072; 
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="email_editar">
         <property name="styleSheet">
          <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 19px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: black; </string>
         </property>
         <property name="text">
          <string>E-mail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="campo_editar_email">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	/*background:transparent;*/
	border:none;
	/*color:#717072;*/
	border-bottom:1px solid #717072; 
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="telefone_editar">
         <property name="styleSheet">
          <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 19px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: black; </string>
         </property>
         <property name="text">
          <string>Telefone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="campo_editar_telefone">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	/*background:transparent;*/
	border:none;
	/*color:#717072;*/
	border-bottom:1px solid #717072; 
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>350</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="cpf_editar">
         <property name="styleSheet">
          <string notr="true">	font-family: &quot;Times New Roman&quot;;
    font-size: 19px;  /* Tamanho da fonte ajustado para um visual mais formal */
    color: black; </string>
         </property>
         <property name="text">
          <string>CPF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="campo_editar_cpf">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	/*background:transparent;*/
	border:none;
	/*color:#717072;*/
	border-bottom:1px solid #717072; 
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
